pack/eoe/eoep.json

```diff
@@ -1033,221 +1670,766 @@
+        "traits": "Item. Tool. Melee.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "08108",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "faction2_code": "mystic",
+        "flavor": "\"Now, for my next trick...\"",
+        "illustrator": "Lin Hsiang",
+        "name": "Etbereal Slip",
+        "pack_code": "eoep",
+        "position": 108,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_willpower": 1,
+        "text": "Choose a non-[[Elite]] enemy at a revealed location up to 2 connections away. Swap places with that enemy.",
+        "traits": "Spell. Trick.",
+        "type_code": "event",
+        "xp": 0
+    },
+    {
+        "code": "08109",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "faction2_code": "mystic",
+        "illustrator": "Andreas Zafiratos",
+        "name": "Blur",
+        "pack_code": "eoep",
+        "position": 109,
+        "quantity": 2,
+        "skill_agility": 1,
+        "slot": "Arcane",
+        "text": "Uses (3 charges).\n[action] If Blur has charges remaining: <b>Evade.</b> For this evasion attempt, you may use [willpower] instead of [agility], and you get +1 skill value. If you succeed, spend 1 charge and you may take an additional action this turn. If you succeed by 0, take 1 damage.",
+        "traits": "Spell.",
+        "type_code": "asset",
+        "xp": 1
+    },
+    {
+        "code": "08110",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "faction2_code": "mystic",
+        "flavor": "\"Now, for my next trick...\"",
+        "illustrator": "Lin Hsiang",
+        "name": "Etbereal Slip",
+        "pack_code": "eoep",
+        "position": 110,
+        "quantity": 2,
+        "skill_agility": 2,
+        "skill_willpower": 1,
+        "text": "Choose a non-[[Elite]] enemy at amy revealed location. Swap places with that enemy.",
+        "traits": "Spell. Trick.",
+        "type_code": "event",
+        "xp": 2
+    },
+    {
+        "code": "08111",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "rogue",
+        "faction2_code": "mystic",
+        "illustrator": "Andreas Zafiratos",
+        "name": "Blur",
+        "pack_code": "eoep",
+        "position": 111,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_willpower": 1,
+        "slot": "Arcane",
+        "text": "Uses (4 charges).\n[action] If Blur has charges remaining: <b>Evade.</b> You may use [willpower] instead of [agility] and get +2 skill value for this evasion attempt. If you succeed, spend 1 or 2 charges and you may take that many additional actions this turn. If you succeed by 0, take 2 damage.",
```

Comment:
Please rejig to read: "instead of [agility], and you get +2 skill value." (missing comma, missing "you", delete final clause).